pandas/indexes/multi.py

```diff
@@ -2102,7 +2204,9 @@ def equals(self, other):
             return True
 
         if not isinstance(other, Index):
-            return False
+            if not isinstance(other, tuple):
+                return False
+            other = Index([other])
```

Comment:
I don't really understand your comment here. 
You mean you want to have `pd.MultiIndex.from_tuples([('a', 1)]).equals(('a', 1))` be True? But why? Also for a single level Index, a 1-length Index does not equals to a scalar with the same value.